docs/source/get_involved/code_of_conduct.rst

```diff
@@ -104,6 +129,11 @@ hope you will consider to be appropriate community guidelines:
    succinct so as to allow maximum engagement by all participants. Do
    not interrupt others on the basis of disagreement; hold such comments
    until they have finished speaking.
+-  **Do not hesitate to voice critical opinions, and accept constructive
```

Comment:
> We can't enforce accepting the constructive criticism

We can outline our expectations.

The python CoC has "gracefully accept constructive criticism", and I think that should be an expectation.

Of course we can enforce that people should accept constructive criticism.

Failure mode: 

"A: I think this code can be improved by changing X to Z."

"B: You have deeply upset me by implying my code needs improvement, or that I have been unable to see that my code needs improvement. This is a clear violation of our code of conduct, and I want you to refrain from hurtful behaviour like this in the future. Please also work on your communication skills."

Reply:
This scenario is exactly what I meant by "fight", moreover, it's a false accusation of CoC violation.
What I mean by "not enforcing" is personal borders, the person doesn't have to change their personal opinion on the matter as long as expression of this opinion doesn't violate CoC. Their PR just not going to be approved and merged - if that's what you mean under "enforce" then I agree.

Reply:
> What I mean by "not enforcing" is personal borders, the person doesn't have to change their personal opinion on the matter as long as expression of this opinion doesn't violate CoC

Yes, I see what you mean. I think what I mean under "enforce" is what you understand, yes.

In common parlance (as far I understand it), "accepting criticism" means acknowledging it without a fight, resistance, or retaliation like above.

It does not mean the recipient has to change their opinion, that would be absurd, because any position can be criticized.